configtx/example_test.go

```diff
@@ -518,11 +522,11 @@ func ExampleConfigTx_SetApplicationMSP() {
 
 // This example shows the addition of a certificate to an orderer org's intermediate
 // certificate list.
-func ExampleConfigTx_SetOrdererMSP() {
+func ExampleUpdatedOrdererOrg_SetMSP() {
 	baseConfig := fetchChannelConfig()
 	c := configtx.New(baseConfig)
 
-	msp, err := c.OrdererMSP("OrdererOrg")
+	msp, err := c.UpdatedConfig().Orderer().Organization("OrdererOrg").MSP()
```

Comment:
It's a little odd for me that, as a user, I'd create a new object from the base config and then have to call `UpdatedConfig` to get the thing I can actually work with.

So, why is that the case?